lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -388,55 +390,54 @@ module Platform
     # Determine what platform we are running on by consulting the metadata
     # service (unless the user has explicitly disabled using that).
     def detect_platform
-      if !@use_metadata_service
-        $log.info "use_metadata_service is false; not detecting platform"
+      unless @use_metadata_service
+        @log.info 'use_metadata_service is false; not detecting platform'
         return Platform::OTHER
       end
 
       begin
         open('http://' + METADATA_SERVICE_ADDR) do |f|
           if (f.meta['metadata-flavor'] == 'Google')
-            $log.info 'Detected GCE platform'
+            @log.info 'Detected GCE platform'
             return Platform::GCE
           end
           if (f.meta['server'] == 'EC2ws')
-            $log.info 'Detected EC2 platform'
+            @log.info 'Detected EC2 platform'
             return Platform::EC2
           end
         end
-      rescue Exception => e
-        $log.debug "Failed to access metadata service: ", :error => e
+      rescue StandardError => e
+        @log.debug 'Failed to access metadata service: ', error: e
       end
 
-      $log.info 'Unable to determine platform'
-      return Platform::OTHER
+      @log.info 'Unable to determine platform'
+      Platform::OTHER
```

Comment:
Yeah, as I said in my other note, I was going to change one thing it would be this - I don't like this syntax at all.  This is something I know is common in perl and I guess ruby adopted it as well.